chain/actions/src/payload.rs

```diff
@@ -70,16 +72,25 @@ pub trait PayloadGenerator<T: Into<TransactionRequest>> {
     /// Creates the transaction payload to immediately close an incoming payment channel
     fn close_incoming_channel(&self, source: Address) -> Result<T>;
 
+    /// Creates the transaction payload to immediately close multiple incoming payment channels
+    fn close_multiple_incoming_channels(&self, sources: Vec<Address>) -> Result<T>;
+
     /// Creates the transaction payload that initiates the closure of a payment channel.
     /// Once the notice period is due, the funds can be withdrawn using a
     /// finalizeChannelClosure transaction.
     fn initiate_outgoing_channel_closure(&self, destination: Address) -> Result<T>;
 
+    /// Creates the transaction payload that initiates the closure of multiple payment channels.
+    fn initiate_multiple_outgoing_channels_closure(&self, destinations: Vec<Address>) -> Result<T>;
+
     /// Creates a transaction payload that withdraws funds from
     /// an outgoing payment channel. This will succeed once the closure
     /// notice period is due.
     fn finalize_outgoing_channel_closure(&self, destination: Address) -> Result<T>;
 
+    /// Creates the transaction payload that withdraws funds from multiple outgoing payment channels.
+    fn finalize_multiple_outgoing_channels_closure(&self, destinations: Vec<Address>) -> Result<T>;
```

Comment:
If possible, the payload generation for channel closure should probably be unified and generate different payloads internally, depending if one or multiple channels were specified.

Reply:
merged with original functions